workflow/commons/src/main/java/com/datastax/oss/dsbulk/workflow/commons/settings/SchemaSettings.java

```diff
@@ -1146,11 +1146,11 @@ private String inferReadQuery(ImmutableMultimap<MappingField, CQLFragment> field
   private void appendTokenRangeRestriction(StringBuilder sb) {
     sb.append(" WHERE ");
     appendTokenFunction(sb);
-    sb.append(" > ");
```

Comment:
Please update the comment in `dsbulk-reference.conf` to reflect above change.

Reply:
Updated and committed the changes.